cli/cmd/lql.go

```diff
@@ -296,14 +302,19 @@ func inputQueryFromURL(url string) (query string, err error) {
 	return
 }
 
-func inputQueryFromEditor(action string) (query string, err error) {
+func inputQueryFromEditor(action string, language string) (query string, err error) {
+	if language != "Rego" && language != "LQL" {
+		err = errors.New("Invalid query language: " + language)
+		return
+	}
 	prompt := &survey.Editor{
 		Message:  fmt.Sprintf("Type a query to %s", action),
 		FileName: "query*.yaml",
 	}
 
 	if (action == "create" || action == "run") && !queryCmdState.EmptyTemplate {
-		prompt.Default = `queryId: YourQueryID
+		if language == "LQL" {
```

Comment:
I wonder if a `switch (language)` would be more idiomatic. The `default` clause might be used to create an error too.

Reply:
Yes switch looks better. Has updated it.